### ISSUE TITLE
SCSS formatter broken

### ISSUE THREAD

**Issue by leaves4j** [octref/vetur#47]
```vue
<template>

</template>

<script>
  // content
</script>

<style lang="scss">
  @import '~styles/main.scss';

  section[role='alert'] {
  @extend .alert;
    @extend .alert-dismissible;
    border-radius: 0;
    margin-bottom: 0;
    background: darken(white, 2);
    border: 0;
    z-index: 1;

    > div {
      @extend .container;
      display: flex;
      flex-direction: row;
      flex-wrap: nowrap;
      justify-content: space-between;
      align-content: stretch;
      align-items: flex-start;

      div {
        order: 0;
        flex: 1 1 auto;
        align-self: auto;
      }
    }

    P:last-of-type {
      @extend .mb-0;
    }
  }
</style>
```
 Format it with shortcut `shift+alt+f`,` scss` are not formatted

**Comment by octref** [comment]
The embedded formatter seems broken. I'll take a look tomorrow.

**Comment by ripper234** [comment]
FYI `LESS` files are affected as well

```
<style lang="less">
#story {
 border: 3px black double;
  padding: 5px;
   border-radius: 15px;
    min-width: 320px;
}
</style>
```

**Comment by octref** [comment]
In 0.4.

![format](https://cloud.githubusercontent.com/assets/4033249/23350217/15fb3462-fc87-11e6-817e-6c4ba6fc3d45.gif)

**Comment by octref** [comment]
Published 0.4 which resolves this problem. Give it a try.

**Comment by ripper234** [comment]
Works great, thanks! 👍
